handler/handler.go

```diff
@@ -60,6 +61,29 @@ func GetTaskInfo(scheduler types.Scheduler) http.HandlerFunc {
 	}
 }
 
+// NotifyCallback handles posting a JSON back to the URI given with the task.
+func NotifyCallback(task *types.EremeticTask) {
+	if len(task.CallbackURI) == 0 {
+		return
+	}
+
+	cbData := task.Status[len(task.Status)-1]
+
+	body, err := json.Marshal(cbData)
+	if err != nil {
+		log.Errorf("Unable to create message for task %s, target uri %s", task.ID, task.CallbackURI)
+	}
+
+	_, err = http.Post(task.CallbackURI, "application/json", bytes.NewBuffer(body))
```

Comment:
I guess this is waiting for the response before returning so potentially a really long wait for a time out here. Should we stick this is in a goroutine to not risk blocking things?